packages/backend/src/server/web/feed.ts

```diff
@@ -39,20 +39,64 @@ export default async function(user: User) {
 	});
 
 	for (const note of notes) {
-		const files = note.fileIds.length > 0 ? await DriveFiles.findBy({
-			id: In(note.fileIds),
-		}) : [];
-		const file = files.find(file => file.type.startsWith('image/'));
+		let contentStr = await noteToString(note, true);
+		let next = note.renoteId ? note.renoteId : note.replyId;
+		let depth = history;
+		while (depth > 0 && next) {
+			const finding = await findById(next);
+			contentStr += finding.text;
+			next = finding.next;
+			depth -= 1;
+		}
+
+		let title = `${author.name} ${(note.renoteId ? 'renotes' : (note.replyId ? 'replies' : 'says'))}`;
+		title += noteintitle ? `: ${note.cw ? note.cw : (note.text ? note.text : 'post a new note')}` : '';
```

Comment:
This still uses a lot of ternary expressions and is hard to read. I think something like this is easier to read:

```suggestion
		let title = author.name + ' ';
		if (note.renoteId) {
		  title += 'renotes';
		} else if (note.replyId) {
		  title += 'replies';
		} else {
		  title += 'says';
		}
		if (noteintitle) {
		  if (note.cw ?? note.text ?? false) {
		    title += ': ' + (note.cw ?? note.text);
		  } else {
		    title += 'something';
		  }
		}
```